print-battleground-state-changes

```diff
@@ -116,6 +116,23 @@ InputRecord = collections.namedtuple(
     ],
 )
 
+
+class VoteDifferential(NamedTuple):
+    amount: int
+    math_string: str
+
+    def __str__(self):
+        return str(self.amount)
```

Comment:
kinda hacky, but needed because we now include `VoteDifferential` in the summary dict which gets serialized into CSV, and want this object to get serialized as just the amount